extensions/Worker.Extensions.ServiceBus/src/Worker.Extensions.ServiceBus.csproj

```diff
@@ -15,8 +15,8 @@
   <Import Project="..\..\..\build\Extensions.props" />
 
   <ItemGroup>
-    <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.16.2" />
-    <PackageReference Include="Microsoft.Extensions.Azure" Version="1.6.3" />
+    <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.17.0" />
+    <PackageReference Include="Microsoft.Extensions.Azure" Version="1.7.1" />
```

Comment:
This change will bring in `Azure.Identity/1.10.3`, which has the major version leap in the downstream dependency. If we don't want that, we can alternatively revert this and explicitly bring in `Azure.Identity/1.10.2`

Reply:
For host repo, we used to 1.10.2 to avoid introducing the breaking change (from 1.10.3)
https://github.com/Azure/azure-functions-host/pull/9646#issuecomment-1785587540